spec/requests/v1/supplemental_claims_controller_request_spec.rb

```diff
@@ -135,6 +139,12 @@ def personal_information_logs
           appeal_submission = AppealSubmission.find_by(submitted_appeal_uuid: id)
           expect(appeal_submission.type_of_appeal).to eq('SC')
           expect { DecisionReview::Form4142Submit.drain }.to change(DecisionReview::Form4142Submit.jobs, :size).by(-1)
+
+          # SavedClaim should be created with request data and list of uploaded forms
+          request_body = JSON.parse(VetsJsonSchema::EXAMPLES.fetch('SC-CREATE-REQUEST-BODY-FOR-VA-GOV').to_json)
+          saved_claim = SavedClaim::SupplementalClaim.find_by(guid: id)
+          expect(saved_claim.form).to eq(request_body.to_json)
```

Comment:
No change necessary, just leaving a note — I wonder if we need to revisit some of these request body examples and add more variations to cover additional test cases 🤔 this one doesn't include an actual upload file, but the NOD example does, so we at least are testing `'nodUploads'` is still persisted in the saved claim, but would be nice if we had a test for `'additionalDocuments'` here too

Reply:
agreed–the more testing scenarios the better! :)